tests/integration/records/test_records_serializers_json.py

```diff
@@ -148,21 +148,58 @@ def test_zero_citations_in_vnd_plus_inspire_record_ui_json(isolated_api_client):
                                        headers={'Accept': 'application/vnd+inspire.record.ui+json'})
     assert response.status_code == 200
     result = json.loads(response.get_data(as_text=True))
-    assert result['metadata']['citations_count'] == 0
+    assert result['metadata']['citation_count'] == 0
 
 
 def test_non_zero_citation_count_in_vnd_plus_inspire_record_ui_json(isolated_api_client):
+    from invenio_db import db
```

Comment:
Ops, `import` in the test :)

Reply:
This is leftover from 'debugging' ;) - will be removed